.travis.yml

```diff
@@ -50,11 +54,51 @@ jobs:
           app: agora-ampad
           env:
             develop: agora-develop
+            DISABLE_OPENCOLLECTIVE: true
+          on:
+            all_branches: true
+          access_key_id: AKIAY22FMBNE7T3Q36PT
+          secret_access_key:
+            secure: "RkCI4mS/yRnMIZp+Wh++DN6gIIy1i1UDQ/V7Vml14pAH9BThurpDJPfEfFZW8twwavJkn5PjPvx4yg3D66o1UQTRGgAVXnYNLpKX9Vr6xhypCD8fNpMR0PVzZJl7JiRpsnWIyjkDbILyqRoF2ACg8IkHSx1gqVkBxMz1gC53DyA9O7rh/xAZvw4Hl/J2tKf+jQSA/Z3JIOWhyn19jkTzTSh5Y5MAX2kZJqDKIbmDaL3fIk/i507Us9mJMIjQVFuP6phtbylqsyuZUe8P2bvNEKrtDAlpNJQIIH6dzCS0KadXIL8LLl3XJ7+U75cB+iNbJFKKr82x2dHHYEQkTOVsdBsMtEDxZXV2TRpryWqG1OsFT11pg0v9OpK+Dhc3a2Oa8SbQQFIrJtYKw2QecQ0vsTaz4U7ro+X+ZUCxk/IneXUxPNOe1OPhBucLHbMoS/x5wvHwW0JzCBAQEEpnFQB9h68JkS5Q+MXvITuXEuly2EROyqPMi55sOkRHJrca1djMmUKqMIIg6yrINhwswth0OPvLsVM0BQRmDF/Ew+7XjbQD2uUvjiukpHjoF/93KRgz/B4ep1f91Kmg2gyu80EJVWWY+m3RWbuz00cO1vZAhvfc+McknsYlA1oSnunXe2MSi+JhO8WuZj5K9jWiI7O88RbXULVf3IJ9JbP+2UT5wu8="
```

Comment:
I assume it's ok to put these secret keys in here, but I feel compelled to confirm that

Reply:
that is not the actual key. it's the secret but encrypted by travis.   only travis knows how to decrypt it.

Reply:
I'll have MattC look at the build error